shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -291,6 +292,15 @@ message DeleteScopeStatus {
     Status status = 1;
 }
 
+message DeleteScopeRecursiveStatus {
+    enum Status {
+        SUCCESS = 0;
+        FAILURE = 1;
+        SCOPE_NOT_FOUND = 2;
```

Comment:
If Scope is not found, our job is done. So we can return success in that case. The option SCOPE_NOT_FOUND is unnecessary.

Reply:
It should be fine to just return a SUCCESS in those cases. I don't see why the user may want to do a different action based on SUCCESS response vs SCOPE_NOT_FOUND response from Controller. We may want to retain the SCOPE_NOT FOUND for backward compatibility though since it already exists as a DELETE_SCOPE_RESPONSE.

Reply:
if user invokes the API for the scope which doesn't exist at all we should not show success, this scope_not_found will be used for that purpose

Reply:
Since its an existing return status. it should be fine to keep it.